cursuri/open-source/curs-1.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: ribbon
+title: "Sisteme de operare și Bash"
+permalink: /cursuri/open-source/curs-1/
+layout: post
+date: 2018-02-19 12:00:00
+author: "Dragos Dimitriu"
+excerpt_separator: <!--more-->
+style: |
+
+    #Cover h2 {
+        margin:30px 0 0;
+        color:#FFF;
+        text-align:center;
+        font-size:70px;
+        }
+    #Cover p {
+        margin:10px 0 0;
+        text-align:center;
+        color:#FFF;
+        font-style:italic;
+        font-size:20px;
+        }
+        #Cover p a {
+            color:#FFF;
+            }
+    #Picture h2 {
+        color:#FFF;
+        }
+    #SeeMore h2 {
+        font-size:100px
+        }
+    #SeeMore img {
+        width:0.72em;
+        height:0.72em;
+        }
+---
+
+# Metodologii de lucru în mediul Open-Source
+
+## Introducere
+
+Cursul are ca scop formarea unui grup de studenți pasionați de tehnologie ce doresc să contribuie la un proiect de anvergură sau ce vor să clădească un nou proiect ce să ajute comunitatea.
+
+## Metoda de evaluare
+
+1. Teme - 5p
```

Comment:
Doar carcotas dar daca lasi doar asta pentru studenti nu cred ca o sa fie suficient ... or sa vrea sa stie cum fac rost de aceste puncte (la fiecare categorie) la ce va ganditi de la proiecte. 
Si daca nu e pe curs aia care nu vin la cursuri o sa faca scandal ca nu e scris si stabilit. 

;)

Reply:
Vor fi mai multe detalii, este _work in progress_ :)